reference/CloudPay-API-Specification.yaml

```diff
@@ -1381,6 +1381,81 @@ paths:
 
         **Note** *Only one of the four authorisation parameters are required*
     parameters: []
+  /api/v1/account/subscriptions:
+    get:
+      operationId: get-api-v1-account-subscriptions
+      summary: Retrieve the authenticated customer's active subscriptions
+      tags:
+        - Account
+      parameters:
+        - $ref: '#/components/parameters/apiJwtTokenParam'
+        - $ref: '#/components/parameters/apiSessionIdParam'
+        - $ref: '#/components/parameters/thirdPartyAuthCookie'
+        - $ref: '#/components/parameters/authCookie'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                description: 'Customer transactions list ordered by createdat ASC'
+                type: array
+                items:
+                  type: object
+                  properties:
+                    SubscriptionName:
```

Comment:
Can you please help me understand the usage of PascalCase for properties instead of camelCase?